README.md

```diff
@@ -36,6 +36,33 @@ To see the graphics, you can open another terminal and run
 GAZEBO_MODEL_PATH=`pwd`/../python/social_bot/models gzclient
 ```
 
+## Environments and Tasks
+
+### [Simple Navigation](python/social_bot/envs/simple_navigation.py)
+
+* [Goal task](python/social_bot/teacher_tasks.py). 
+    
+    To be updated
+
+### [Grocery Ground](python/social_bot/envs/grocery_ground.py)
+
+* [Goal task](python/social_bot/teacher_tasks.py). 
+
+    <img src="examples/grocery_ground_pioneer.gif" width="320" height="240" alt="pioneer"/>
```

Comment:
images should be not be under directory examples. In ALF, they are under examples because they are actually the examples. We can create a folder "media" to put these pictures.

Reply:
yes. Moved to "media".